src/loaders/relocate-loader.js

```diff
@@ -558,28 +557,26 @@ module.exports = function (code) {
             transformed = true;
             magicString.overwrite(staticChildNode.start, staticChildNode.end, replacement);
           }
-          staticChildNode = staticChildValue = undefined;
         }
-        else if (stats && stats.isDirectory() &&
-            // dont emit __dirname or package base
-            staticChildValue !== path.dirname(id) &&
-            staticChildValue !== pkgBase) {
```

Comment:
This is the way in which directory emission is being widened - we were previously not emitting packages or __dirname references at all.

What this means is that code as simple as - `console.log(__dirname)` will cause the current directory to be emitted.